gradle.properties

```diff
@@ -3,15 +3,15 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
     # check these on https://modmuss50.me/fabric.html
-    minecraft_version=1.19.3
-    yarn_mappings=1.19.3+build.5
+    minecraft_version=1.19.4
+    yarn_mappings=1.19.4+build.2
     loader_version=0.14.21
 
 # Mod Properties
-    mod_version=1.5.2-release
+    mod_version=1.5.3-release
     maven_group=net.reikeb.notenoughgamerules
-    archives_base_name=NEG-FABRIC-1.19.3
+    archives_base_name=NEG-FABRIC-1.19.4
 
 # Dependencies
     # check this on https://modmuss50.me/fabric.html
-    fabric_version=0.76.1+1.19.3
+    fabric_version=0.85.0+1.19.4
```

Comment:
Could you modify the `.github/ISSUE_TEMPLATE/fabric_bug_report.yml` to add this new fabric version?

Reply:
Ofc